package.json

```diff
@@ -64,7 +64,7 @@
     "ts-mocha": "^9.0.2"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.0.0"
```

Comment:
why the change to the minimum node version?

Reply:
https://oclif.io/blog/2022/01/12/announcing-oclif-v2#node--12